tsconfig.json

```diff
@@ -7,8 +7,6 @@
     { "path": "packages/ai/openai" },
     { "path": "packages/cli" },
     { "path": "packages/cluster" },
-    { "path": "packages/cluster-browser" },
-    { "path": "packages/cluster-node" },
```

Comment:
When we deprecate these, let's push a deprecation notice to npmjs.com for these packages. And also do that for the other packages we have deprecated in the past (especially @effect/schema, which is still seeing more downloads than it should).

Reply:
They are being rewritten in another branch